core/store/migrate/migrations/0215_functions_subscriptions.sql

```diff
@@ -0,0 +1,19 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE functions_subscriptions(
+    router_contract_address bytea,
+    subscription_id bigint,
+    owner bytea CHECK (octet_length(owner) = 20) NOT NULL,
+    balance text,
+    blocked_balance text,
```

Comment:
Can these two fields be of type BIGINT? LINK token has 18 decimals so it should be fit.